DSharpPlus/Entities/Guild/DiscordRoleTags.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+    /// <summary>
+    /// Represents a discord role tags.
+    /// </summary>
+    public class DiscordRoleTags
+    {
+        /// <summary>
+        /// Gets the id of the bot this role belongs to.
+        /// </summary>
+        [JsonProperty("bot_id", NullValueHandling = NullValueHandling.Ignore)]
+        public ulong? BotID { get; internal set; }
```

Comment:
The convention is `Id` like in `DiscordMessage.ChannelId`

Reply:
oops, thanks for spotting that!